src/no_panic.rs

```diff
@@ -1,8 +1,8 @@
 /// A wrapper around a slice that exposes no functions that can panic.
 ///
-/// Intentionally avoids implementing `PartialEq` and `Eq` to avoid implicit
-/// non-constant-time comparisons.
-#[derive(Clone, Copy, Debug)]
+/// Intentionally avoids implementing `Debug`, `Eq`, and `PartialEq` to avoid
+/// creating a side channel that would leak information about the value.
+#[derive(Clone, Copy)]
```

Comment:
Hmm...I would expect that with this PR and the recently-merged PRs, that Codecov would see this as fully-covered. Regardless, this PR doesn't actually decrease coverage here.